lib/src/main/java/org/exoplatform/social/addons/updater/RDBMSMigrationManager.java

```diff
@@ -140,6 +156,10 @@ public void run() {
                       && !MigrationContext.isIdentityDone()) {
                 getIdentityMigrationService().start();
                 updateSettingValue(MigrationContext.SOC_RDBMS_IDENTITY_MIGRATION_KEY, MigrationContext.isIdentityDone());
+
+                if (useMigrationIdentityStorage && MigrationContext.isIdentityDone()) {
+                  identityManager.setIdentityStorage(identityStorage);
```

Comment:
Shouldn't we put this in a finally block to make sure we come back to the default implementation in case something goes wrong ?